src/redux/actions/postActions.js

```diff
@@ -8,10 +10,13 @@ import {
   DELETE_POST,
   EDIT_POST,
   GET_PAGES,
-  GET_PAGES_NEXT
+  GET_PAGES_NEXT,
+  REPORT_POST,
+  GET_REPORTS,
+  DELETE_REPORT
 
 } from '../constants/types';
-import { postArticleApi, getPostsApi, getSingleArticle, deleteArticleApi, editArticle, getPagesApi, getNextPageApi } from '../../services/api';
+import { postArticleApi, getPostsApi, getSingleArticle, deleteArticleApi, editArticle, getPagesApi, getNextPageApi,reportArticleApi,getreportsApi,deleteReportApi } from '../../services/api';
```

Comment:
Hello @Amoswachira ,
You could put the imports in new lines instead of having all of them in a long line

Reply:
@Ogutu-Brian okay

Reply:
@Ogutu-Brian it is fixed